randovania/games/prime1/pickup_database/pickup-database.json

```diff
@@ -135,7 +135,8 @@
             "broad_category": "beam_related",
             "model_name": "Wave Beam",
             "offworld_models": {
-                "am2r": "sItemWaveBeamPrime"
+                "am2r": "sItemWaveBeamPrime",
+                "planets_zebeth": "spr_ITEM_Wave_Beam"
```

Comment:
IIRC dunc or toaster didn't like it if beams were only partially represented. I.e. if no plasma beam exists, you should probably remove the entries for wave and ice.